### ISSUE TITLE
e.component not a function?

### ISSUE THREAD

**Issue by robbiemu** [vuejs-tips/vue-the-mask#26]
Im trying to use this in my component to mask mm:ss and transform numeric input automagically. I installed with `npm i -S vue-the-mask`:

```
+ vue-the-mask@0.11.0-alpha.2
added 114 packages and updated 1 package in 22.598s
```

then I edited my component:

```vue
<template>
  <masked-input name="scene_length" mask="##:##" :masked="true" type="text" class="form-control" placeholder="mm:ss" v-model="newScene.duration">
    </masked-input>
<!-- ... -->
<script>
import TheMask from 'vue-the-mask'
//...
export default {
  components: { 'masked-input': TheMask },
//...
```
That produces this error:
```
vue.esm.js:479 [Vue warn]: Error in render function: "TypeError: e.component is not a function"

found in

---> <AddScene> at C:\Users\roberto\Qwire\github\dev2\qwireclear-frontend\src\components\wbf\scene\scenes_overview\AddScene.vue
       <ScenesOverview> at C:\Users\roberto\Qwire\github\dev2\qwireclear-frontend\src\components\wbf\scene\ScenesOverview.vue
         <App> at C:\Users\roberto\Qwire\github\dev2\qwireclear-frontend\src\components\App.vue
           <Root>
warn @ vue.esm.js:479
handleError @ vue.esm.js:562
Vue._render @ vue.esm.js:4130
updateComponent @ vue.esm.js:2538
get @ vue.esm.js:2881
Watcher @ vue.esm.js:2870
mountComponent @ vue.esm.js:2542
Vue$3.$mount @ vue.esm.js:7863
Vue$3.$mount @ vue.esm.js:10066
init @ vue.esm.js:3501
createComponent @ vue.esm.js:5147
createElm @ vue.esm.js:5090
patch @ vue.esm.js:5606
Vue._update @ vue.esm.js:2424
updateComponent @ vue.esm.js:2538
get @ vue.esm.js:2881
run @ vue.esm.js:2958
flushSchedulerQueue @ vue.esm.js:2724
(anonymous) @ vue.esm.js:701
nextTickHandler @ vue.esm.js:648
vue.esm.js:566 TypeError: e.component is not a function
    at r (vue-the-mask.js:1)
    at resolveAsyncComponent (vue.esm.js:2109)
    at createComponent (vue.esm.js:3587)
    at _createElement (vue.esm.js:3797)
    at createElement (vue.esm.js:3736)
    at vm._c (vue.esm.js:4076)
    at Proxy.render (AddScene.vue?6af9:79)
    at VueComponent.Vue._render (vue.esm.js:4128)
    at VueComponent.updateComponent (vue.esm.js:2538)
    at Watcher.get (vue.esm.js:2881)
```

If I go on to change to the directive form it works:
```vue
<template>
  <input name="scene_length" v-mask="'##:##'" :masked="true" type="text" class="form-control" placeholder="mm:ss" v-model="newScene.duration">
    </input>
<!-- ... -->
<script>
import { mask } from 'vue-the-mask'
//...
export default {
  directives: { mask },
//...
```

**Comment by neves** [comment]
Try:
`import {TheMask} from 'vue-the-mask'`
I'll fix the docs.

**Comment by neves** [comment]
Fixed with version v0.11.0-beta
